onadata/apps/main/urls.py

```diff
@@ -38,6 +38,7 @@
     re_path('^api/v1/', include(router.urls)),
     re_path('^api/v1/', include(router_with_patch_list.urls)),
     re_path(r'^service_health/$', service_health),
+    re_path(r'^service_health/minimal/$', service_health_minimal, name='service-health-minimal'),
```

Comment:
you can probably use `path()` instead of `re_path()`, like you did in https://github.com/kobotoolbox/kpi/pull/4763/files#diff-e085d48b36e6c4a6ca57a37cc5b0da8c1c663439e5589e13d4077aadfede7311R8

Reply:
Hey @RuthShryock, sorry I neglected this. Thanks for making changes. With `path()`, though, you'd no longer need a regular expression (so the `^` and `$` anchors should be removed).

TBH I'd be surprised (and interested!) to learn if the `path()` function works when given a regular expression :)

Reply:
The url worked fine on my end with the `path()` function receiving a regular expression but I updated it just to be safe :)